src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -111,10 +152,20 @@ internal async Task ProcessMessageAsync(Message message, CancellationToken cance
             }
 
             FunctionResult result = await _triggerExecutor.ExecuteAsync(message, cancellationToken);
-
             await _messageProcessor.CompleteProcessingMessageAsync(message, result, cancellationToken);
         }
 
+        internal async Task ProcessSessionMessageAsync(IMessageSession session, Message message, CancellationToken cancellationToken)
+        {
+            if (!await _sessionMessageProcessor.BeginProcessingMessageAsync(session, message, cancellationToken))
```

Comment:
See ServiceBusTriggerExecutor.PopulateTriggerDetails - I think it would be good to add SessionId to the details we log.